### ISSUE TITLE
some questions about synthetic training data

### ISSUE THREAD

**Issue by bachw433** [atapour/monocularDepth-Inference#1]
Hi Atapour,

thanks for sharing. the result is really amazing!!
But I'm not really sure whether what I understood about synthetic data is correct:

By exploiting the tool _DeepGTAV_, you put a camera on the visual car in GTA for data collection.
So that you can get training data from that camera's perspective.

Then, I'm wondering how you get the ground truth disparity. 
Did you put two cameras on the car for triangulation calculation? 
Could I have the training datasets you used for training? or just some sample pairs of data with ground truth. 

Secondly, why not to train on depth directly instead of disparity, so that the model could directly output the depth?

thanks

**Comment by atapour** [comment]
Hi,
DeepGTAV provides virtual disparity (not from two cameras) by default and further post processing is needed to get the real depth. Real depth can subsequently be used for training.

**Comment by bachw433** [comment]
Cool! It seems that DeepGTAV can do more than I imagined. I'll try it.

But I'm still really confused about training ground truth.
which ground truth did you used for training, disparity or depth?
Is there any advantage of it?

**Comment by atapour** [comment]
In this code, disparity is used as the ground truth. But I don't see why depth couldn't be used since depth and disparity are easily convertible to one another.

Either depth or disparity can be used as ground truth for training. The reason why disparity is used here is because that is what was provided by default.

**Comment by bachw433** [comment]
okay, thanks.
cause I'm thinking, since disparity and depth are in reciprocal relationship.
If we take disparity as ground truth, close range part (namely, high disparity) will be more precise than far distance part (low disparity).
on the other hand, if the depth is used as ground truth, far distance part would be more precise than close range part.
Do you agree?

**Comment by yipengm** [comment]
Did you use DeepGTAV from https://github.com/aitorzip/DeepGTAV/issues/98 to generate GTA data? I tried the DeepGTAV but cannot find the disparity information

**Comment by yipengm** [comment]
> Hi,
> DeepGTAV provides virtual disparity (not from two cameras) by default and further post processing is needed to get the real depth. Real depth can subsequently be used for training.

Did you use DeepGTAV from https://github.com/aitorzip/DeepGTAV to generate GTA data? I tried the DeepGTAV but cannot find the disparity information
